### PR TITLE
12.0 mig base_search_fuzzy

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Please remove the warning:
```
2019-01-10 15:51:21,489 152 WARNING openerp_test odoo.models: The model trgm.index has no _description
```

File: base_search_fuzzy/tests/test_query_generation.py [hunk 1]
```diff
@@ -74,7 +74,8 @@ def test_fuzzy_search(self):
             return
 
         if not self.TrgmIndex.index_exists('res.partner', 'name'):
-            field_partner_name = self.env.ref('base.field_res_partner_name')
```
Comment (pedrobaeza) [inline base_search_fuzzy/tests/test_query_generation.py]: FYI, this is now `base.field_res_partner__name`

**Review by pedrobaeza — COMMENTED** [review]
Please squash together these commit for enhancing commit history:

seleccion_024

**Review by cubells — APPROVED** [review]
:+1: 

LGTM

Code review and test

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**